### PR TITLE
little silly changes to docstrings

### PR DESCRIPTION
**Pull request by mmagnuski**
I had this file open in the browser and found a few things that could be changed. :boom:

### CODE REVIEW

File: mne/viz/utils.py [hunk 1]
```diff
@@ -99,8 +99,7 @@ def tight_layout(pad=1.2, h_pad=None, w_pad=None, fig=None):
 
 
 def _check_delayed_ssp(container):
-    """ Aux function to be used for interactive SSP selection
-    """
+    """Aux function to be used for interactive SSP selection"""
```
Comment (jona-sassenhagen) [inline mne/viz/utils.py]: .

File: mne/viz/utils.py [hunk 2]
```diff
@@ -110,7 +109,7 @@ def _check_delayed_ssp(container):
 
 
 def _validate_if_list_of_axes(axes, obligatory_len=None):
-    """ Helper function that validates whether input is a list/array of axes"""
+    """Helper function that validates whether input is a list/array of axes"""
```
Comment (jona-sassenhagen) [inline mne/viz/utils.py]: .

**Comment by agramfort** [comment]
if you (we?) want to do this then let's do it completely using

https://pypi.python.org/pypi/pep257

**Comment by Eric89GXL** [comment]
Oooh that looks like a fun project. I'm on it unless someone else wants to

**Comment by mmagnuski** [comment]
Go for it @Eric89GXL :) I can close this PR, despite it's unquestionable awesomeness. ;P

**Comment by Eric89GXL** [comment]
I'll cherry-pick your commit as part of another PR

**Pull request closed without merging**